src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -1805,7 +1805,7 @@ void VM_Version::get_processor_features() {
   }
 
   // Allocation prefetch settings
-  intx cache_line_size = prefetch_data_size();
+  int cache_line_size = checked_cast<int>(prefetch_data_size());
```

Comment:
I don't see why the old code would produce a warning, except with -Wsign-conversion on 32-bit platforms.

Reply:
I put the cast in because prefetch_data_size() returns uint and I'm narrowing it to int which has a lower positive range.  I assume checked_cast would find that.

Reply:
OK, so you caught it by inspection and are fixing it proactively :-)

Reply:
Yeah heading off these -Wsign-conversion warnings :)